app/lib/messages.js

```diff
@@ -7,15 +7,20 @@ function mandatorySelectionMessage() {
 }
 
 function emptyPostcodeMessage() {
-  return 'You must enter a postcode to find the service you are looking for.';
+  return 'You must enter a postcode.';
 }
 
 function invalidPostcodeMessage(location) {
-  return `We can't find the postcode '${location}'. Check the postcode is correct and try again.`;
+  return `We can't find the postcode '${location.toUpperCase()}'. Check the postcode is correct and try again.`;
 }
 
 function outsideOfEnglandPostcodeMessage() {
-  return 'This is an England only service. Please enter an English postcode.';
+  /* eslint-disable quotes */
+  return `This postcode is not in England. Get help to find a chlamydia test in \
```

Comment:
Rather than having the html in the code file, can this be moved into the view? Perhaps use a flag on the response object to determine if it should be displayed or not.

Reply:
The error message pattern is well defined, and both the integration and unit tests have been added. Adding the flag will break that making the view more complicated. I'm hoping to leave it like this unless you feel very strongly about it.

Reply:
Whilst the pattern of having a simple string is in place, I'd say this should not be case when there is html elements being included within the message. There is going to be reduced or no support for static help such as syntax highlighting when the html is contained in a JS file compared to when it is within a view file. I'd like to see it moved into a view please.